### PR TITLE
fix link to DO README.md

### PR DESCRIPTION
**Pull request by stephenlauck**

### CODE REVIEW

**Review by trevorghess — APPROVED** [review]
👍 I'd suggest we establish a contributors guide and maintainers guide. I'd say spelling fixes don't need approval from when submitted by maintainers and can be merged without review. I'd suggest we add a tag for spelling-fix or something just to track.

**Review by irvingpop — APPROVED** [review]

**Pull request merged**
